tests/unit-ext/test-ext-arg.c

```diff
@@ -115,7 +115,7 @@ test_validator1_handler (const jerry_value_t func_obj_val __attribute__((unused)
     TEST_ASSERT (arg1);
     TEST_ASSERT (arg2 == 10.5);
     TEST_ASSERT (arg3[0] == 'a' && arg3[1] == 'b'
-                 && arg3[2] == 'c' && arg3[4] == '\0');
+                 && arg3[2] == 'c' && arg3[3] == '\0');
```

Comment:
if we are already touching this, why not using `strcmp (arg3, "abc")`? it might be more descriptive and does exactly the same, if I'm not mistaken